gateway/handlers/update_handler.go

```diff
@@ -1,16 +1,130 @@
 package handlers
 
 import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
 	"net/http"
+	"time"
 
 	"github.com/alexellis/faas/gateway/metrics"
+	"github.com/alexellis/faas/gateway/requests"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/swarm"
 	"github.com/docker/docker/client"
 )
 
-// MakeUpdateFunctionHandler request to update an existing function with new configuration such as image, parameters etc.
+const updateConfigROLLBACK = 2 // From https://github.com/docker/swarmkit/blob/master/api/types.proto#L357
```

Comment:
is there a comment for this line that would help it make sense?

Reply:
I'll update, was just pointing to the source of the magic value

Could pull in the swarmkit types as a dependency, probably should be doing that in retrospect.

Reply:
ah, its not actually needed, hangover from an early attempt at getting the rollback working, I've deleted